### ISSUE TITLE
Different scollSpeed for different section

### ISSUE THREAD

**Issue by SahilMepani** [lukehaas/Scrollify#333]
Is it possible to get different scollSpeed for different sections.

`$.scrollify({
    section : '.scroll-section',
    scrollSpeed: 1400,
    easing: 'linear',
    updateHash: false,
    setHeights: false,
  });`

I want certain .scroll-section have different .scroll-speed.

**Comment by lukehaas** [comment]
Yes, use the `setOptions` method to change the scroll speed based on the index of the current section.
